<!--
  1 failure:  :disappointed_rel...
  0 warning: 
  1 messages
  
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@7.2.0 test /Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9986-OOPS7OUYJ46UM5XGUGPVKMS5UDEL3HTO6IDQSPCOVLBQSIYJS77Q
> grunt

Running "appcJs:src" (appcJs) task

Running "eslint:src" (eslint) task

/Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9986-OOPS7OUYJ46UM5XGUGPVKMS5UDEL3HTO6IDQSPCOVLBQSIYJS77Q/build/scons-modules-integrity.js
  9:12  warning  Found non-literal argument in require  security/detect-non-literal-require

/Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9986-OOPS7OUYJ46UM5XGUGPVKMS5UDEL3HTO6IDQSPCOVLBQSIYJS77Q/android/cli/commands/_buildModule.js
  554:18  warning  Found non-literal argument to RegExp Constructor  security/detect-non-literal-regexp

/Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9986-OOPS7OUYJ46UM5XGUGPVKMS5UDEL3HTO6IDQSPCOVLBQSIYJS77Q/tests/Resources/ti.ui.tableview.addontest.js
  11:5   warning  'should' is assigned a value but never used    no-unused-vars
  15:3   warning  'didFocus' is assigned a value but never used  no-unused-vars
  33:1   error    Trailing spaces not allowed                    no-trailing-spaces
  33:2   error    Mixed spaces and tabs                          no-mixed-spaces-and-tabs
  36:2   error    Mixed spaces and tabs                          no-mixed-spaces-and-tabs
  37:2   error    Mixed spaces and tabs                          no-mixed-spaces-and-tabs
  38:2   error    Mixed spaces and tabs                          no-mixed-spaces-and-tabs
  41:40  error    Missing space before function parentheses      space-before-function-paren

✖ 10 problems (6 errors, 4 warnings)
  2 errors, 0 warnings potentially fixable with the `--fix` option.

Warning: Task "eslint:src" failed. Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

--- a/tests/Resources/ti.ui.tableview.addontest.js
+++ b/tests/Resources/ti.ui.tableview.addontest.js
@@ -8,7 +8,6 @@
 /* global Ti */
 /* eslint no-unused-expressions: 'off' */
 'use strict';
-var should = require('./utilities/assertions');
 
 describe('Titanium.UI.TableView', function () {
 	var win,
@@ -30,15 +29,16 @@ describe('Titanium.UI.TableView', function () {
 	it('appendSection and appendRow (TIMOB-25936)', function (finish) {
 		win = Ti.UI.createWindow({ backgroundColor: '#f00' });
 		var table = Ti.UI.createTableView();
-		 
+
 		for (var i = 0; i < 2; ++i) {
 			table.appendSection(Ti.UI.createTableViewSection({ headerTitle: 'Header ' + i, className: 'Header' }));
-		    for (var j = 0; j < 3; j++) {
-		    	table.appendRow(Ti.UI.createTableViewRow({ title: 'Row ' + j, className: 'Row' }));
-		    }
+			for (var j = 0; j < 3; j++) {
+				table.appendRow(Ti.UI.createTableViewRow({ title: 'Row ' + j, className: 'Row' }));
+			}
 		}
 
-		win.addEventListener('open', function() {
+		win.addEventListener('open', function () {
+			didFocus = true;
 			finish();
 		});
 

